nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/AdjustContrast.java

```diff
@@ -0,0 +1,19 @@
+package org.nd4j.linalg.api.ops.custom;
+
+import org.nd4j.base.Preconditions;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.ops.DynamicCustomOp;
+
+public class AdjustContrast extends BaseAdjustContrast {
+
+    public AdjustContrast() {super();}
+
+    public AdjustContrast(INDArray in, double factor, INDArray out) {
+        super(in, factor, out);
+    }
+
+    @Override
+    public String opName() {
```

Comment:
You should also define tensorflow opname. There's method for that.

Reply:
Added.

Reply:
Done.